### ISSUE TITLE
Error when starting GitHawk without internet connection

### ISSUE THREAD

**Issue by j-f1** [GitHawkApp/GitHawk#2596]
My bookmarks are migrated, but I get this error when starting GitHawk without an internet connection:

![GitHawk Upload by j-f1](https://i.imgur.com/vF1cuRj.jpg)

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.27.1 (1546622353)
Device: iPhone 8 (iOS 12.1.2)
TestFlight: true
</pre>
</details>

**Comment by rnystrom** [comment]
Can’t seem to repro this. Is it persistent if you go back online/offline again?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Yup did that, just got a general error message, not a migration alert

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by j-f1** [comment]
Huh, are you on TF?

**Comment by Huddie** [comment]
I got a squawk no internet connection error 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
@j-f1 yup, same here @Huddie 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by j-f1** [comment]
Looks like the current TestFlight is the prod version, so I can’t enable FLEX.

**Comment by j-f1** [comment]
Seems like there’s a GET request to the GraphQL API, then a POST to fetch a bunch of repos (maybe other stuff too — the query is so long it exceeds the horizontal-scroll abilities of FLEX)

**Comment by gingerbeardman** [comment]
I had this in #2598. I had cellular data but no wifi at the time, though it was a weak signal.

This was the first time I was aware a migration had or was happening.
